third_party/move/move-compiler-v2/src/bytecode_generator.rs

```diff
@@ -266,6 +267,7 @@ impl<'env> Generator<'env> {
             | Operation::MoveFrom
             | Operation::MoveTo
             | Operation::Abort
+            | Operation::Vector
```

Comment:
Why do we need an Operation::Vector now?

Reply:
Because `vector[...]` is a new expression primitive which wasn't handled here before.